File: Java/2.3.1-完全切分/src/text.java
import com.hankcs.hanlp.corpus.io.IOUtil;
import com.hankcs.hanlp.dictionary.CoreDictionary;

import java.io.IOException;
import java.util.TreeMap;

public class text
{
    public static void main(String[] args) throws IOException {//读取词典
        TreeMap<String, CoreDictionary.Attribute> dictionary
                = IOUtil.loadDictionary("Y:/NLP/Anaconda/Install/Lib/site-packages/pyhanlp/static/data/dictionary/CoreNatureDictionary.mini.txt");
        System.out.println(Main.segmentFully("我和杜玉洁就读于北京信息科技大学", dictionary));

    }
}
//text 待分词的文本
//dictionary 词典，一定要先加载词典
//return 单词列表

File: Python/2.3.1-完全切分/text.py

from pyhanlp import *

from Main import fully_segment


def load_dictionary():
    '''
    加载HanLP中的mini库
    :return: 一个set语句
    '''

    IOUtil = JClass('com.hankcs.hanlp.corpus.io.IOUtil')#利用JClass取得了HanLP中的IOUtil类
    path = HanLP.Config.CoreDictionaryPath.replace('.txt', '.mini.txt')#取得了HanLP中的词典路径
    dic = IOUtil.loadDictionary([path])#调用了IOUtil类的静态方法
    return set(dic.keyset())

dic = load_dictionary()
print(fully_segment('项目的研究', dic))